includes/utilities.inc

```diff
@@ -462,11 +462,19 @@ function islandora_solr_islandora_basic_collection_backend_callable($collection_
   $qp = new IslandoraSolrQueryProcessor();
   $_islandora_solr_queryclass = $qp;
 
-  $qp->buildQuery(format_string('!member_field:("info:fedora/!pid" OR "!pid") OR !collection_member_field:("info:fedora/!pid" OR "!pid")', array(
-    '!member_field' => variable_get('islandora_solr_member_of_field', 'RELS_EXT_isMemberOf_uri_ms'),
-    '!pid' => $collection_object->id,
-    '!collection_member_field' => variable_get('islandora_solr_member_of_collection_field', 'RELS_EXT_isMemberOfCollection_uri_ms'),
-  )), drupal_get_query_parameters());
+  $qp->buildQuery("*:*", drupal_get_query_parameters());
+  $pid = $collection_object->id;
+  $member_of = variable_get('islandora_solr_member_of_field', 'RELS_EXT_isMemberOf_uri_ms');
+  $member_of_collection = variable_get('islandora_solr_member_of_collection_field', 'RELS_EXT_isMemberOfCollection_uri_ms');
+  $filters = array(
```

Comment:
I think your format_string should _not_ be in an array, I dropped this on my dev site and I get every item back in every collection. It appears that in the `fq` parameter this one is nested too deep. Probably just `$filters = format_string(....` should fix it.

Reply:
@whikloj totally! My bracket paranoia commit (https://github.com/Islandora/islandora_solr_search/pull/331/commits/2d55546df02a9a71bcc056ca42373d9603ef728d) is the responsible (not me!). Fixing that, so sorry